web/src/routes/index.svelte

```diff
@@ -3,13 +3,36 @@
 </svelte:head>
 
 <script>
+    	import Typewriter from 'svelte-typewriter'
 </script>
-
-<div class="mt-4 md:columns-4">
-    <div class="flex"><div class="text-9xl">1.</div><div class="h-full text-4xl">Basic details</div></div>
-    <div class="flex"><div class="text-9xl">2.</div><div class="h-full text-4xl">Tweet it with photos</div></div>
-    <div class="flex"><div class="text-9xl">3.</div><div class="h-full text-4xl">Click start</div></div>
-    <div class="flex"><div class="text-9xl">4.</div><div class="h-full text-4xl">Collect sats</div></div>
+<div class="flex justify-center">
+<div class="mt-4 md:columns-3 w-3/5">
+    <div class="flex"><div class="text-7xl lg:text-9xl">1.</div><div class="h-full text-2xl lg:text-4xl">Tweet Photos</div></div>
+    <div class="flex"><div class="text-7xl lg:text-9xl">2.</div><div class="h-full text-2xl lg:text-4xl">Click Start</div></div>
+    <div class="flex"><div class="text-7xl lg:text-9xl">3.</div><div class="h-full text-2xl lg:text-4xl">Collect Sats</div></div>
+</div>
+</div>
+<div class="flex justify-center">
+<div class="mt-12 columns-1 md:columns-3 md:w-11/12">
+    <span>
+        <p class="flex-row text-s md:text-xl lg:text-3xl lg:text-right">Sell anything...</p>
+    </span>
+    <span class="flex bg-black/30">
+<Typewriter interval={[50, 60, 95, 80]} cursor='255,255,255,30' loop>
+        <span class="text-s md:text-xl lg:text-3xl">
+            <p>1 hour of your time</p>
+            <p>bitcoin art</p>
+            <p>your furniture</p>
+            <p>an ASIC miner</p>
+            <p>Baklava</p>
```

Comment:
definitely lowercase!

Reply:
actually... can baklava even be auctioned?

Reply:
I had included baklava (and capitalized it!) because that's how they were listed in issue #11 
Any input on this, @chiefmonky ? Can someone really auction Baklava? and should it be capitalized? :)